src/jit/instrsxarch.h

```diff
@@ -27,24 +24,28 @@
   #error Unexpected target type
 #endif
 
-#ifndef INST1
-#error  At least INST1 must be defined before including this file.
+#ifndef INSTRUCTION
```

Comment:
INSTRUCTION should be `#undef`ed at the end of this file, so includers don't need to do that.

Reply:
Fixed.